src/game/propobj.c

```diff
@@ -5218,7 +5218,10 @@ void hovTick(struct defaultobj *obj, struct hov *hov)
 	f32 ymax;
 	f32 ymin;
 
-	if (g_Vars.lvframe60 > hov->prevframe60) {
+#ifdef PLATFORM_N64
+	if (g_Vars.lvframe60 > hov->prevframe60)
+#endif
+	{
```

Comment:
For some reason which I don't understand yet, when the combat boost is enabled g_Vars.lvframe60 == hov->prevframe60 every other frame, causing the transform to be updated every other frame therefore the jittering. I'll keep investigating this.